src/lib/components/button/__snapshots__/button.test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`Button is backwards compatible (using text prop instead of children) 1`
   className="theme-base"
 >
   <div
-    className="container "
+    className=""
```

Comment:
We should delete this className if it is not necessary

Reply:
this is automatically generated – it's a test. So, button has a container class name prop, but if it's not set, it will be empty.

Reply:
haha my bad I didn't realised it was the snapshot hahahaha xD